tests/tests/include/UtilsTest.php

```diff
@@ -38,22 +38,47 @@
  * display the words "Powered by SugarCRM" and "Supercharged by SuiteCRM".
  */
 
-
 namespace SuiteCRM;
 
 if (!defined('sugarEntry') || !sugarEntry) {
     die('Not A Valid Entry Point');
 }
 
-/**
- * StateCheckerCestAbstract
- *
- * Implementation of state checker Codeception tests.
- *
- * @author SalesAgility
- */
-abstract class StateCheckerCestAbstract
+
+use SuiteCRM\StateCheckerPHPUnitTestCaseAbstract;
+
+include_once __DIR__ . '/../../../include/utils.php';
+
+class UtilsTest extends StateCheckerPHPUnitTestCaseAbstract
```

Comment:
please use `SuiteCRM\StateCheckerUnitAbstract` as parent class since we using codeception.